pootle/templates/editor/units/edit.html

```diff
@@ -152,19 +140,7 @@
                 </div>
                 {% endif %}
               </div>
-              <div id="js-unit-{{ unit.id }}"
-                   class="translate-original{% if unit.hasplural %} translate-plural{% endif %}">
-                {% for i, source, title in unit|pluralize_source %}
-                <div
-                  class="translation-text js-translation-text"
-                  lang="{{ source_language.code }}"
-                  dir="{{ source_language.direction }}"
-                  {% if title %} title="{{ title }}"{% endif %}
-                  data-string="{{ source|escapejs }}"
```

Comment:
@julen I hit an encoding issue when I set `data-string` attribute in UnitSource.
I'll try again now and let you know what the issue is.